docs/docs/05-configuration.md

```diff
@@ -155,3 +162,48 @@ The short form of a full URL string is enough for general use. For advanced conf
 `on` is a special property for setting event handler functions on proxy server events. See the section on ["Listening for Proxy Events"](https://github.com/http-party/node-http-proxy#listening-for-proxy-events) for a list of all supported events. You must be using a `snowpack.config.js` JavaScript configuration file to set this.
 
 This configuration has no effect on the final build.
+
+#### Mount Options
+
+
+```js
+// snowpack.config.json
+{
+  "mount": {
+    // Files in the local src directory are written to `/_dist_/*` in the final build.
+    "src": "/_dist_",
+    // Files in the local public directory are written to `/*` in the final build.
+    "public": "/"
+  }
+}
+```
+
+The `mount` configuration lets you map local files to their location in the final build. If no mount configuration is given, then the entire current working directory (minus excluded files) will be built and mounted to the Root URL (Default: `/`, respects `baseUrl`).
+
+#### Alias Options
+
+> Note: In an older version of Snowpack, all mounted directories were also available as aliases by default. As of Snowpack 2.7, this is no longer the case and no aliases are defined by default.
+
+```js
+// snowpack.config.json
+{
+  alias: {
+    // Type 1: Package Import Alias
+    "lodash": "lodash-es",
+    "react": "preact/compat",
+    // Type 2: Local Directory Import Alias (relative to cwd)
+    "components": "./src/components"
+    "@app": "./src"
+  }
+}
+```
+
+The `alias` config option lets you define an import alias in your application. When aliasing a package, this allows you to import that package by another name in your application. This applies to imports inside of your dependencies as well, essentially replacing all references to the aliased package. 
+
+Aliasing a local directory (any path that starts with "./") creates a shortcut to import that file or directory. While we don't necessarily recommend this pattern, some projects do enjoy using these instead of relative paths:
+
+```diff
+-import '../../../../../Button.js';
++import '@app/Button.js';
+```
```

Comment:
👍🏻 Love `diff`s wherever possible

Reply:
<img width="658" alt="Screen Shot 2020-07-28 at 3 48 09 PM" src="https://user-images.githubusercontent.com/1369770/88725767-d6ec7700-d0e9-11ea-86aa-4ab73c696ecd.png">

ack the coloring is reversed!